target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -171,15 +171,19 @@ f9k1115v2)
 	ucidef_set_led_usbdev "usb2" "USB2" "belkin:green:usb2" "1-1"
 	;;
 
-rb-750)
-	ucidef_set_led_default "act" "act" "rb750:green:act" "1"
+rb-750 | rb-751)
+	ucidef_set_led_timer "act" "act" "rb750:green:act" "1000" "1000"
 	ucidef_set_led_netdev "port1" "port1" "rb750:green:port1" "eth1"
 	ucidef_set_led_switch "port2" "port2" "rb750:green:port2" "switch0" "0x10"
```

Comment:
the default behaviour has always been that flashing means booting and/or failsafe, changing this will confuse people i think. please remove this timer trigger part.